resources/windows/wix/project.wxs.erb

```diff
@@ -6,7 +6,7 @@
     Name="<%= settings[:product_name] %>"
     Language="1033"
     Codepage="1252"
-    Version="<%= @version.sub(/\.g[0-9a-z]{7}$/, '') %>"
+    Version="<%= @platform.windows_style_version(@version) %>"
```

Comment:
I am going to throw up a maint commit to change the name to make this clear it's for WiX